packages/components/src/UICompatibilityView/UICompatibilityViewCommon.js

```diff
@@ -1,34 +0,0 @@
-// @flow
-import React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
-
-import { UIStyle, UITextStyle } from '@tonlabs/uikit.core';
-
-import { uiLocalized } from '@tonlabs/localization';
-
-import UIComponent from '../UIComponent';
-
-const styles = StyleSheet.create({
-    container: {
-        justifyContent: 'center',
-        alignItems: 'center',
-    },
-});
-
-export default class UICompatibilityViewCommon extends UIComponent<null, null> {
-    renderNotCompatible() {
-        return (
-            <Text style={UITextStyle.primaryBodyRegular}>
-                {uiLocalized.WeAreSorryButYourBrowserVersionIsNotCompatible}
```

Comment:
If we don't need it anymore, you can clean up this translation as well

Reply:
@sertony done!